src/Command/Project/ProjectCreateCommand.php

```diff
@@ -98,6 +99,27 @@ protected function execute(InputInterface $input, OutputInterface $output)
             try {
                 $organization = $this->validateOrganizationInput($input, 'create-subscription');
```

Comment:
Hello, we are negotiating in Slack about possible changes to the HAL logic here so maybe there won't be a NoOrganizationsException